source/includes/stackpath/_origin_settings.md

```diff
@@ -80,10 +74,7 @@ Attributes | &nbsp;
 `origin.password` <br/> *string* | Password to use when authenticating with the origin. 
 `origin.sslValidationEnabled` <br/> *boolean* | Specifies if SSL validation for the origin is enabled. 
 `origin.commonCertificateName` <br/> *string* | Common name to validate SSL origin requests against.
-`origin.path` <br/> *string* | The origin's HTTP request path.
-`origin.hostname` <br/> *string* | The origin's HTTP request hostname. Can be a valid IPv4 address or a valid domain name. 
-`origin.port` <br/> *int* | The origin's HTTP port. Must be one of [80, 8080, 443, 1935, 9091].
-`origin.securePort` <br/> *int* | The origin's HTTPS port. Must be one of [80, 8080, 443, 1935, 9091].
+`origin.address` <br/> *string* | The address of the primary origin that the CDN uses to pull content from. Can be a valid IPv4 address or a valid domain name. It may include a specific port and a precise path as well (e.g. 199.250.204.212:80/test). 
```

Comment:
We should still document the list of valid ports

Reply:
a408e2f